pallets/runtime/src/asset.rs

```diff
@@ -255,8 +268,8 @@ decl_storage! {
         /// (ticker) -> SecurityToken details [returns SecurityToken struct]
         pub Tokens get(fn token_details): map Ticker => SecurityToken<T::Balance>;
         /// Used to store the securityToken balance corresponds to ticker and Identity
-        /// (ticker, DID) -> balance
-        pub BalanceOf get(fn balance_of): map (Ticker, IdentityId) => T::Balance;
+        /// (ticker, DID) -> AssetBalance
+        pub BalanceOf get(fn balance_of): double_map hasher(blake2_256) Ticker, blake2_256(IdentityId) => AssetBalance<T::Balance>;
```

Comment:
Question: If we invert the double-map key order, we could easily iterate over all tickets are owned by an specific Identity. That case could be needed by UI.
I am not sure if I am missing the use-case for order (ticker,DID).
WDYT @maxsam4 ?

Reply:
The current requirements are for the token issuer to be able to generate cap tables for their tokens. i.e. Given a ticker, it should be possible to list all token holders of that ticker and their balances. This is why the ordering of keys is as it is. If we get an additional requirement that a user should be able to iterate over all the tokens they hold then we'll have to make changes.